src/Views/FormPages/PersonalInfo.jsx

```diff
@@ -102,15 +102,15 @@ function PersonalInfo(props){
             <Checkbox id="prev-name-change" aria-describedby="prev-name-change_alert" name="prev-name-change" data-test="checkBox" checked={props.previousName} onChange={props.onChangePreviousName} label={stringContent.nameChange} />
         )}
 
-        <div id="prev-name-change_alert" className="usa-alert usa-alert--info" role="region" aria-live="polite">
+        <div id="prev-name-change_alert" className="usa-alert usa-alert--info" role="region" aria-live="polite" aria-label="informative alert">
             <div className="usa-alert__body">
                 <div className="usa-alert__text" dangerouslySetInnerHTML={{__html: nameSectionAlert}}/>
             </div>
         </div>
 
-        <h3 className={'margin-top-5'}>{nameSectionField.label}</h3>
+        <Fieldset legend={<h3 className={'margin-top-5 margin-bottom-0'}>{nameSectionField.label}</h3>} className="fieldset">
```

Comment:
**Question**
1. why was the margin-bottom-0 added?  did adding the fieldset adjust the style or something?
2. why was the className added as well? i could be wrong but i didnt see/remember needing that for this

Reply:
1. Yeah, it seems like adding the Fieldset adjusted the spacing a little bit -- specifically added a bit more space between "What is your legal name?" and "Using the name..." than was there originally. This is what it looks like without the margin-bottom-0:
![image](https://github.com/user-attachments/assets/fd8d68c0-5bc1-42fa-8827-926a0b8bf05c)
and with (matches how it originally looked):
![image](https://github.com/user-attachments/assets/d4078eb6-7d78-4ed9-81ff-78f9195226f7)
 I'm guessing that this is a result of some of the default styling in the Fieldset USWDS component.

2. I think that the className was there before -- this is what the line looked like before this:
![image](https://github.com/user-attachments/assets/6d636be0-3496-41ff-89bf-4f0e96897444)